### PR TITLE
Bump versions of ndk to 0.6, ndk-sys to 0.3, ndk-glue to 0.6

### PR DESCRIPTION
**Pull request by enfipy**
- [x] Tested on all platforms changed
- [x] Added an entry to `CHANGELOG.md` if knowledge of this change could be valuable to users
- [x] Updated documentation to reflect any user-facing changes, including notes of platform-specific behavior
- [x] Created or updated an example program if it would help users understand this functionality
- [x] Updated [feature matrix](https://github.com/rust-windowing/winit/blob/master/FEATURES.md), if new features were added or implemented

### CODE REVIEW

**Review by msiglreith — COMMENTED** [review]
Thanks, please update the version section in the readme.

**Comment by enfipy** [comment]
> Thanks, please update the version section in the readme.

Sure, any plans on releasing a new patch version of winit? Or place it as part of `winit v0.26`?

**Comment by madsmtm** [comment]
These are breaking changes (because `ndk` is a public dependency), and should be noted as such in the changelog (and we'll have to release it as part of `winit v0.27.0`)

File: CHANGELOG.md [hunk 1]
```diff
@@ -10,6 +10,7 @@ And please only add new entries to the top of this list, right below the `# Unre
 
 - Remove no longer needed `WINIT_LINK_COLORSYNC` environment variable.
 - **Breaking:** Rename the `Exit` variant of `ControlFlow` to `ExitWithCode`, which holds a value to control the exit code after running. Add an `Exit` constant which aliases to `ExitWithCode(0)` instead to avoid major breakage. This shouldn't affect most existing programs.
+- Bump `ndk` version to 0.6, ndk-sys to `v0.3`, `ndk-glue` to `0.6`.
```
Comment (madsmtm) [inline CHANGELOG.md:13]: Please place this at the top of the list (see the above text)

**Comment by enfipy** [comment]
@madsmtm Updated as you requested.

**Review by madsmtm — APPROVED** [review]
Cool, thanks!

**Review by msiglreith — APPROVED** [review]

**Pull request merged**